org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/filepath/FilePathSettingsForTest.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+* Copyright (c) 2023 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lemminx.extensions.filepath;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.lemminx.extensions.filepath.settings.FilePathExpression;
+import org.eclipse.lemminx.extensions.filepath.settings.FilePathMapping;
+import org.eclipse.lemminx.extensions.filepath.settings.FilePathSupport;
+import org.eclipse.lemminx.extensions.filepath.settings.FilePathSupportSettings;
+
+/**
+ * {@link FilePathSupportSettings} for tests.
+ * 
+ */
+public class FilePathSettingsForTest {

```

Comment:
if it's a helper class, don't give it a Test suffix, as JUnit will try to run it

Reply:
renamed